.ci/jenkins_job_build.sh

```diff
@@ -58,11 +58,17 @@ pr_number=
 
 if [ -n "$pr_number" ]
 then
-	# For PRs we rebase the PR commits onto the defined target branch
-	git fetch origin "pull/${ghprbPullId}/head" && git checkout master && git reset --hard FETCH_HEAD && git rebase origin/${ghprbTargetBranch}
+	pr_branch="PR_${pr_number}"
+
+	# Create a separate branch for the PR. This is required to allow
+	# checkcommits to be able to determine how the PR differs from
+	# "master".
+	git fetch origin "pull/${pr_number}/head:${pr_branch}"
+	git checkout "${pr_branch}"
+	git rebase "origin/${ghprbTargetBranch}"
 else
 	# Othewise we test the master branch
-	git fetch origin && git checkout master && git reset --hard origin/master
+	git fetch origin && git checkout master && git reset --hard "origin/${ghprbTargetBranch}"
```

Comment:
No you cannot do that, the variable `${ghprbTargetBranch}` won't be defined for a master build. This is only a valid variable for PRs.

Reply:
Good catch - fixed!